src/ex5/server/db/migrations/20220622202408-create-items.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Items', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      item_id: {
```

Comment:
the model already contains an id column out-of-the-box
it's impressing that you made another column and took care of id generation, it works, but you could use the default id instead

Reply:
also the name is inconsistent with the other fields - should have been itemId